modules/Forum/classes/Forum.php

```diff
@@ -604,17 +604,35 @@ public function getPosts($tid = null){
 		}
 		return false;
 	}
-
+	
+	// Converting Cyrillic to Latin letters (https://en.wikipedia.org/wiki/ISO_9)
```

Comment:
Could this be moved to the file core/classes/Util.php and made static please? Would be easier for other modules to access this function 👍